TheThingsNetwork/lorawan-stack
# Fix scroll jumps when selecting tabs of a table

#### Summary
This quickfix introduces react-router's `<ScrollRestauration />` component to the root route, which takes better care of restoring the scroll position after navigation changes. This resolves unnecessary scroll jumps when navigating.

#### Changes



- Replace our `<ScrollToTop />` utility component with `<ScrollRestauration />` and some custom logic for determining the scroll key
- Remove `scroll-to-top.js` utility

#### Testing

1. Navigate to the Packet Broker
2. Toggle between the `Networks` and `Default routing policy`-tab
3. Observe the scroll position staying (relatively) at the same position (as opposed to the page scrolling back to the top)

#### Notes for Reviewers
This will solve annoying back-to-top scrolling when selecting tabs that connect to the navigation. This is especially noticeable in the Packet Broker settings panel.

#### Checklist
